compiler/damlc/daml-visual/src/DA/Daml/Visual.hs

```diff
@@ -173,8 +173,12 @@ startFromExpr seen world e = case e of
         Set.singleton (AExercise tpl (LF.ChoiceName "Archive"))
     -- NOTE(RJR): Look for calls to the `exercise` method from a `Choice`
     -- instance and produce the corresponding edge in the graph.
+    --(LF.ETmApp (LF.ETyApp (LF.ETyApp (LF.ETyApp (EInternalTemplateVal "exercise") (LF.TCon tpl)) (LF.TCon (LF.Qualified _ _ (LF.TypeConName [chc])))) _ret), _dict)
```

Comment:
I don’t find this comment super helpful. If you don’t understand pattern synonyms it’s not clear what this is supposed to be and if you do, then inlining the definition here in a comment somewhat defeats the point of using a pattern synonym. Maybe just add a comment that this is a pattern synonym with a link to docs? Or probably better at the definition of `EInternalTemplateVal` since finding that just entails grep.

Reply:
Ah sorry I left that in. This is not about pattern synonyms (as you can see the synonym is still used in there). This is because I printed the syntax tree then reconstructed the expected shape (which is the comment), then inverted it to match the existing style of infix operators.